### ISSUE TITLE
radio not working in Safari

### ISSUE THREAD

**Issue by ismael7** [lusaxweb/vuesax#378]
everything is selected
<img width="1599" alt="screen shot 2018-11-22 at 10 13 50 pm" src="https://user-images.githubusercontent.com/32073211/48928385-f4445900-eea3-11e8-8a45-dac65411525f.png">

**Comment by brunokunace** [comment]
yes,  vuesax dont have today support for all browser's (aka IE and Safari)

**Comment by michalszajnecki** [comment]
From https://github.com/lusaxweb/vuesax  **README.md**
```
Browser Support
Recent versions of Firefox, Chrome, Edge, Opera and Safari. IE10+
```

**Comment by MagdielCAS** [comment]
Isn't working on chrome also...

**Comment by luisDanielRoviraContreras** [comment]
@ismael7 You could show me the console, sure there is some error that can help solve it, thank you very much for the contribution

@MagdielCAS You could pass me the version and more information to be able to fix the error

**Comment by damuso** [comment]
this has been already fixed
